app/noScript.js

```diff
@@ -77,7 +81,7 @@ function onHeadersReceived (details) {
   }
 
   let csp = details.responseHeaders[cspHeaderName] || []
-  csp.push("script-src 'none'")
+  csp.push('script-src chrome-extension://*')
```

Comment:
it also breaks noscript UI in the bravery panel (hard coded to count for console errors involving script-src none). removing for now